haystack/schema.py

```diff
@@ -9,10 +11,10 @@
 class Document:
     def __init__(
         self,
-        text: str,
+        text: str, # change text to content/data to cope for images, tables ...?
```

Comment:
let's go for "content" and add content_type (text, image, table)

Reply:
In the case of tables, content will be of type pd.dataframe. 
We will need to make sure about a proper json representation of it in a) REST API and b) Pipelines (for allowing cross node HTTP communication between nodes)